libs/core/algorithms/include/hpx/parallel/algorithms/for_loop_induction.hpp

```diff
@@ -232,7 +232,7 @@ namespace hpx::experimental {
 }    // namespace hpx::experimental
 
 namespace hpx::parallel { inline namespace v2 {
-
+    /// \cond NOINTERNAL
```

Comment:
These will most likely create conflicts with #5939 (once it's merged). Could you resolve those conflicts, please?

Reply:
Done!